scripts/update-associations.js

```diff
@@ -0,0 +1,132 @@
+/*eslint-disable */
+/**
+ *
+ * This script queries dfp for all line items that match the arguments
+ * specified, modifies their javascript representation and the submits an update
+ * to DFP.
+ *
+ * Usage:
+ *
+ *   $ node scripts/update-associations.js --channel A --platform M --position MIDDLE --region USA --partner SONOBI
+ *
+ */
+'use strict';
+
+var Bluebird = require('bluebird');
+var argv = require('minimist')(process.argv.slice(2));
+var _ = require('lodash');
+
+var DFP_CREDS = require('../local/application-creds');
+var config = require('../local/config');
+var formatter = require('../lib/formatter');
+
+var Dfp = require('node-google-dfp-wrapper');
+
+var credentials = {
+  clientId: DFP_CREDS.installed.client_id,
+  clientSecret: DFP_CREDS.installed.client_secret,
+  redirectUrl: DFP_CREDS.installed.redirect_uris[0]
+};
+
+var dfp = new Dfp(credentials, config, config.refreshToken);
+
+// read command line arguments
+var creativeId = argv.creativeId;
+
+var ProgressBar = require('progress');
+var progressBar;
+
+var CONCURRENCY = {
+  concurrency: 1
+};
+
+var exampleAssociation = {
+  lineItemId: null,
+  creativeId: null,
+  startDateTime: {
+    date: { year: 2016, month: 2, day: 9 },
+    hour: 11,
+    minute: 19,
+    second: 35,
+    timeZoneID: 'America/New_York'
+  },
+  startDateTimeType: 'USE_START_DATE_TIME',
+  sizes: null,
+  status: null,
+  stats: {stats:{}},
+  lastModifiedDateTime: null
+};
```

Comment:
- [ ] This part is in here because we need to have an object to force certain properties to be placed in the right order (specifically `sizes`). I can add a comment.